substrate/client/rpc-spec-v2/src/lib.rs

```diff
@@ -49,7 +49,7 @@ pub enum MethodResult {
 impl MethodResult {
 	/// Constructs a successful result.
 	pub fn ok(result: impl Into<String>) -> MethodResult {
-		MethodResult::Ok(MethodResultOk { success: true, result: result.into() })
+		MethodResult::Ok(MethodResultOk { success: true, value: result.into() })
```

Comment:
is this only used by the archive API?
why not move it to the archive module so it doesn't get used by something else?

Reply:
yes, these are only used by the archive APIs

Reply:
https://github.com/paritytech/polkadot-sdk/issues/7982